### PR TITLE
Community page implemented #6418

### PR DESCRIPTION
**Pull request by gautampanday**
At Community page basic layout completed

### CODE REVIEW

File: theme/templates/ecommerce.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {% extends 'base.html' %}
-{% block title %} {% trans %} ECommerce {% endtrans %} | {{ super() }} {% endblock title %}
+{% block title %} {% trans %} NEWS {% endtrans %} | {{ super() }} {% endblock title %}
```
Comment (tarunbhardwaj) [inline theme/templates/ecommerce.html]: News? Changes should only be for community template

File: theme/templates/community.html [hunk 2]
```diff
@@ -0,0 +1,242 @@
+{% extends 'base.html' %}
+{% block title %} Community | {{ super() }} {% endblock title %}
```
Comment (tarunbhardwaj) [inline theme/templates/community.html]: trans

File: theme/templates/community.html [hunk 3]
```diff
@@ -1,65 +1,241 @@
...
+             </div>
+             <div class="col-sm-12 col-md-4 text-center">
+               <i class="fa fa-envelope-o"></i>
+             </div>
+           </div>
+           <div class="mar_top2"></div>
+           <div class="divider-blue"></div>
+           <div class="mar_top2"></div>
+           <p>{% trans %}Feel free to subscribe to our{% endtrans %} <a href="#">{% trans %}mailing list:{% endtrans %}</a></p>
+           <form role="form">
```
Comment (sharoonthomas) [inline theme/templates/community.html:52]: This is not feasible

**Comment by sharoonthomas** [comment]
Closing this pr too.. I think this design needs to be rethoght

**Pull request closed without merging**